SU-HSDO/suhumsci
# Bugherd tweaks

# READY FOR REVIEW

# Summary
- Change margin setting from global card mixin; update to only add to views
- Remove margin above headings in paragraph items for first children
- Center heading dash for headings that are center aligned
- Remove heading dash from headings in footer
- Fix box-shadow issue on masonry item and clean up masonry code
- Global/general margin and spacing tweaks

# Needed By (Date)
- End of sprint (7/19)

# Urgency
- How critical is this PR?

# Steps to Test

1. Pull this branch
2. Ensure that various tweaks are reflected

# Affected Projects or Products
- Archaeology
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-153
- https://stanfordits.atlassian.net/browse/ARCH-154
- https://stanfordits.atlassian.net/browse/ARCH-164
- https://stanfordits.atlassian.net/browse/ARCH-165
- https://stanfordits.atlassian.net/browse/ARCH-172
- https://stanfordits.atlassian.net/browse/ARCH-175
- https://stanfordits.atlassian.net/browse/ARCH-183

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
